TwoDicePig/src/TwoDicePig.java

```diff
@@ -68,24 +68,23 @@ private static int playerTurn(String name, int idx) {
 				System.out.println("First die value: " + die1);
 				System.out.println("Second die value: " + die2);
 
+				// If two 1s are rolled, the player�s entire score is lost, and the turn ends.
 				if (die1 == 1 && die2 == 1) {
-					// If two 1s are rolled, the player�s entire score is lost, and the turn ends.
 					totalScore = 0;

```

Comment:
This can be removed too, since the totalScore variable isn't used elsewhere within the scope of this if block